code/datums/systems/ds/battle_royale.dm

```diff
@@ -1,33 +1,32 @@
-/// Global list of areas which are considered to be inside the same department for our purposes
-GLOBAL_LIST_INIT(battle_royale_regions, list(
-	"Medical Bay" = list(
-		/area/station/command/heads_quarters/cmo,
-		/area/station/medical,
-		/area/station/security/checkpoint/medical,
-	),
-	"Research Division" = list(
-		/area/station/command/heads_quarters/rd,
-		/area/station/security/checkpoint/science,
-		/area/station/science,
-	),
-	"Engineering Bay" = list(
-		/area/station/command/heads_quarters/ce,
-		/area/station/engineering,
-		/area/station/maintenance/disposal/incinerator,
-		/area/station/security/checkpoint/engineering,
-	),
-	"Cargo Bay" = list(
-		/area/station/cargo,
-		/area/station/command/heads_quarters/qm,
-		/area/station/security/checkpoint/supply,
-	),
-))
-
-/// Basically just exists to hold references to datums so that they don't GC
+
 DATASYSTEM_DEF(battle_royale)
 	name = "Battle Royale"
 	/// List of battle royale datums currently running
 	var/list/active_battles
+	/// Static list of available regions that are considered the same department
+	var/static/list/regions = list(list(
```

Comment:
(it doesn't _really_ need to be static since the battle royale datasystem is a singleton but i don't know if it matters or not)

Reply:
them being static technically makes init time worse. but it's such a minor minor nit that it doesn't really matter